website/docs/os.getSystemTags.md

```diff
@@ -0,0 +1,19 @@
+Returns the list of tags corresponding to a system.
+
+There are tags specific to each operating system (see [system()](system.md) for a complete list of identifiers.),
+and meta tags like `posix`, `darwin`, `desktop` and `mobile` tags.
+
+### Examples ###
+```lua
+print("iOS system tags: " .. os.getSystemTags("ios"))
+-- { "ios", "darwin", "posix", "mobile" },
```

Comment:
Won't this print `iOS system tags: <table 0x...>` or something similar? I think this is a good idea for an example otherwise.